packages/node/src/checkin.ts

```diff
@@ -0,0 +1,33 @@
+import type { CheckIn, CheckInEvelope, CheckInItem, DsnComponents, SdkMetadata } from '@sentry/types';
+import { createEnvelope, dsnToString } from '@sentry/utils';
+
+/**
+ * Create envelope from check in item.
+ */
+export function createCheckInEnvelope(
+  checkIn: CheckIn,
+  metadata?: SdkMetadata,
+  tunnel?: string,
+  dsn?: DsnComponents,
+): CheckInEvelope {
+  const headers: CheckInEvelope[0] = {
```

Comment:
l: can we reuse this function for header creation? 

https://github.com/getsentry/sentry-javascript/blob/449fb87df54e2b3abbaaf72e334add5894a3364b/packages/utils/src/envelope.ts#L232

Although I'm realizing that this adds more headers than we need here and we probably need to refactor the utility function. I'm also totally fine if we leave it as is.

Reply:
yup, let me take a look at it in a future PR.